lib/user_stats.rb

```diff
@@ -27,4 +27,30 @@ def self.list_user_domains(start_date=nil, limit=nil)
     User.connection.select_all(sql)
   end
 
+  # Returns the number of domant users for the given domain
+  # (A dormant user is one with no requests, tracks or comments)
+  def self.count_dormant_users(domain, start_date=nil)
+    # When we have Rails 4 across the board, we get to say "where.not" and
+    # rewrite this using the ORM
+    # example code here: http://stackoverflow.com/a/23389130), until then...
+    #
+    # Reminder - check that the returned ids in the subquery does not include
```

Comment:
As far as it can be, reworded as the note for future maintainers that I think my brain intended to send